content/pt-br/docs/concepts/overview/working-with-objects/object-management.md

```diff
@@ -0,0 +1,180 @@
+| Comandos imperativos              | Objetos ativos       | Projetos em desenvolvimento | 1+                 | Baixo         |
+| Configuração do objeto imperativo  | Arquivos individuais     | Projetos em produção    | 1                  | Moderado       |
+| Configuração da declaração do objeto | Diretórios de arquivos | Projetos em produção    | 1+                 | Alto           |
+
+## Comandos imperativos
+
+Ao usar comandos imperativos, um usuário opera diretamente em 
+objetos ativos em um *cluster*. O usuário fornece operações para
+o comando `kubectl`  como argumentos ou *flags*.
+
+Isso é a maneira recomendada para iniciar ou executar uma única tarefa
+em um *cluster*. Pois esta técnica opera diretamente em objetos ativos,
+não fornecendo uma histórico de configurações anteriores.
+
+### Exemplos
+
+Execute uma instância do *nginx container* criando um objeto de *deploy*:
+
+```sh
+kubectl create deployment nginx --image nginx
+```
+
+### Comparações
+
+Vantagens comparadas às configurações de objetos:
+
+- Os comandos são expressos como uma única palavra de ação .
+- Os comandos exigem um único passo para realizar mudanças no *cluster*.
+
+Desvantagens comparadas às configurações de objetos:
+
+- Os comandos não integram com o processo de revisão de mudanças. 
+- Os comandos não fornecem uma trilha de auditoria associada a mudanças.
+- Os comandos não fornecem uma fonte de registros, exceto o que está ativo.
+- Os comandos não fornecem um *template* para a criação de novos objetos.
+
+## Configuração do objeto imperativo
+
+Na configuração do objeto imperativo o comando kubectl especifica a operação
+(criar, substituir, etc.), *flags* opcionais e pelo menos uma arquivo de nomes. 
+O arquivo especificado deve conter uma definição completa do objeto em formato
+YAML ou JSON. 
+
+Veja a [referência da API ](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+para mais detalhes sobre as definições de objetos.
+
+{{< warning >}}
+O comando imperativo `replace` substitui a especificação exstente
+coom uma nova, descartando todas as alterações do objeto ausente do
+arquivo de configuração. Essa abordagem não deve ser utilizada com 
+tipos de recursos cujas especificações são atualizadas independentemente 
+do arquivo de configuração. Serviços do tipo `LoadBalancer`, por exemplo, 
+possuem seu campo `externalIPs` atualizado independente da configuração 
+do cluster.
+{{< /warning >}}
+
+### Exemplos
+
+Crie os objetos definidos no arquivo de configuração: 
+
+```sh
+kubectl create -f nginx.yaml
+```
+
+Exclua os objetos definidos em dois arquivos de cofiguração:
+
+```sh
+kubectl delete -f nginx.yaml -f redis.yaml
+```
+
+Atualize os objetos definidos em um arquivo de configuração substituindo
+a configuração ativa:
+
+```sh
+kubectl replace -f nginx.yaml
+```
+
+### Comparações 
+
+Vantagens comparadas à comandos imperativos:
```

Comment:
There should be no grave accent (à) before a masculine and plural word. I'd say "Vantagens comparadas aos comandos imperativos"